scripts/advanced_pdf_generator.py

```
@@ -0,0 +1,150 @@
+            print(f"PDF generated successfully: {output_pdf}")
+            return output_pdf
+            
+    except Exception as e:
+        print(f"Error generating PDF: {e}")
+        return None
+    finally:
+        # Clean up the temporary file
+        try:
+            os.remove(temp_file)
+        except:
+            pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)